.github/workflows/pull-request.yml

```
@@ -58,7 +58,11 @@ jobs:
       - name: Check for linting errors in modified files
         if: steps.changed-files.outputs.only_changed != 'true'
         env: 
+<<<<<<< HEAD
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 61: could not find expected ':'

(syntax-check)